exercises/zh/exc_01_03_02.py

```diff
@@ -1,15 +1,19 @@
-# 导入英文语言类并创建nlp对象
+# 导入中文语言类并创建nlp对象
 from ____ import ____
 
 nlp = ____
 
 # 处理文本
-doc = ____("I like tree kangaroos and narwhals.")
+doc = ____("我喜欢老虎和狮子。")
 
-# 截取Doc中"tree kangaroos"的部分
-tree_kangaroos = ____
-print(tree_kangaroos.text)
+# 遍历打印doc中的内容
+for i in doc:
+    print(i.text)
```

Comment:
I would use `token` as the loop variable here, since this is most consistent with how we normally do it in the docs etc. (and makes it clear that it's a token object, not an index etc).

```suggestion
for token in doc:
    print(token.text)
```

Also, I assume you added this loop here as a reminder for the user that shows how the tokenization works?

Reply:
Yes using token is indeed better.
I will do it as
```
for i, token in enumerate(doc):
    print(i, token.text)
```
The reason is that in some exercises user need to count the token position in the sentence, and since Chinese tokenization is not by character but by words so it is not trival to read that order without printing it.